sdk/python/feast/infra/utils/snowflake_utils.py

```diff
@@ -53,13 +53,20 @@ def get_snowflake_conn(config, autocommit=True) -> SnowflakeConnection:
     else:
         kwargs = {}
 
+    if "schema" in kwargs.keys():
+        kwargs["schema_"] = kwargs.pop("schema")
+
     kwargs.update((k, v) for k, v in config_dict.items() if v is not None)
     [
         kwargs.update({k: '"' + v + '"'})
         for k, v in kwargs.items()
         if k in ["role", "warehouse", "database", "schema_"]
     ]
-    kwargs["schema"] = kwargs.pop("schema_")
+
+    if "schema" in kwargs.keys():
+        kwargs["schema"] = kwargs.pop("schema_")
+    else:
+        kwargs["schema"] = '"PUBLIC"'
```

Comment:
Any reason why this is happening here as well as on line 56?

Reply:
schema is a protected word so I need to do whacky stuff without forcing users to set schema_ as opposed to a more friendly schema

Reply:
Oops - i understand this snippet now, I misread it earlier. Makes sense.